### DISCUSSION TITLE
Is there anyway/event where I can get the column size for each block present in the screen

### DISCUSSION THREAD

**Discussion by groot432** [katoid/angular-grid-layout#46] [Ideas]
Is there anyway/event where I can get the column size for each block present in the screen ,In a way i can enhance apply more feature into it,
Thanks in advance

**Comment by llorenspujol** [comment]
Right now there isn't any API, but you can calculate yourself doing something like:

```typescript
const gridClientRect = (this.gridElementRef.nativeElement as HTMLElement).getBoundingClientRect();
const colWidth = clientRect.width / this.cols;
```
